subquery/src/mappings/nodl.ts

```diff
@@ -64,9 +64,15 @@ export async function handleERC20Transfer(event: TransferLog): Promise<void> {
       value
     );
 
-    from.balance =
+    const fromNewBalance =
       (from.balance || BigInt(0)) - (value > 0 ? value : BigInt(0));
```

Comment:
I'd rather get error on unexpected values than having an assumption for them. A value < 0 should produce error. If you add that error then you can change this line:
```suggestion
    const fromNewBalance =
      (from.balance || BigInt(0)) - value;
```